docs/changelog.md

```diff
@@ -30,6 +30,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
   more details.
 - `six-file-upload`: added `code` attribute to `SixFileUploadFailurePayload`, which specifies the
   type of error that has occurred
+- New `six-rating` form component
```

Comment:
If you could add an example of the six-rating to the react or angular app that would be cool. :)

Reply:
works for me :)